### PR TITLE
1.2 desktop optimize

### PR DESCRIPTION
**Pull request by CodyyLee**

### CODE REVIEW

**Review by Gabe0214 — APPROVED** [review]
merge conflicts should be resolved

File: frontend/src/App.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ import ArchivedDeckView from './components/archivedDeckView/archivedDeckView';
 import EditCard from './components/cards/EditCard';
 import NewCard from './components/cards/NewCard';
 import DeckImport from './components/decks/DeckImport';
+import SearchDeck from "./components/searchDeck/searchDeck";
```
Comment (GCJ2) [inline frontend/src/App.js:29]: This is imported twice.
Please see line 16 for reference.
Thank you.

**Review by GCJ2 — APPROVED** [review]
Thank you for your attention to the double import.

**Review by Gabe0214 — APPROVED** [review]
merge conflicts should be resolved for real this time.

**Pull request merged**